scapy/layers/radius.py

```diff
@@ -567,9 +571,11 @@ def compute_message_authenticator(radius_packet, packed_req_authenticator,
                                       shared_secret):
         """
         Computes the "Message-Authenticator" of a given RADIUS packet.
+        (RFC 2869 - Page 33)
         """
 
-        data = prepare_packed_data(radius_packet, packed_req_authenticator)
+        data = prepare_packed_data(radius_packet,
+                                   packed_req_authenticator, True)
```

Comment:
As this is the only place where you call this with `True`, it would probably be much cleaner to handle the `value` here.
You'll get rid of the warning

Reply:
Thanks, great suggestion. Check the new commit then. I believe now everything is OK.
I'm doing everytime force push to my repository to not pollute scapy with this minor commits.